www_src/pages/editor/text-editor.jsx

```diff
@@ -0,0 +1,93 @@
+var React = require('react/addons');
+var render = require('../../lib/render.jsx');
+var Binding = require('../../lib/binding.jsx');
+
+var Range = require('../../components/range/range.jsx');
+var ColorGroup = require('../../components/color-group/color-group.jsx');
+var {CheckboxSet, Radio} = require('../../components/option-panel/option-panel.jsx');
+
+var TextEditor = React.createClass({
+  mixins: [React.addons.LinkedStateMixin, Binding],
+  getInitialState: function () {
+    return {
+      fontSize: 18,
+      fontFamily: 'Roboto',
+      color: '#645839',
+      fontWeight: 'normal',
+      fontStyle: 'normal',
+      textDecoration: 'none',
+      textAlign: 'center'
+    };
+  },
+  render: function () {
+    var textStyleOptions = [
```

Comment:
Again, requires a decision on the AST stuff etc that is outside the scope of this patch.